flax/linen/module.py

```diff
@@ -421,7 +421,8 @@ def __init_subclass__(cls):
     # automatically transform Modules into dataclasses at subclass creation
     # time, and we set the last dataclass arguments to `parent` and `name`.
     cls._customized_dataclass_transform()
-    # We wrap user-defined methods including setup and __call__ to enforce
+    # We wrap user-defined methods including setup and _
```

Comment:
?

Reply:
Good catch — undoing whatever that was. Thanks @avital !

```suggestion
    # We wrap user-defined methods including setup and __call__ to enforce
```